tests/cluster/cosmos/test_comos.py

```diff
@@ -20,7 +20,14 @@ class CosmosTests(unittest.TestCase):
     def test_01_tables(self):
```

Comment:
the file name has a misspelling "tests/cluster/cosmos/test_comos.py" -> "tests/cluster/cosmos/test_cosmos.py"